pkgs/by-name/tc/tcld/package.nix

```diff
@@ -0,0 +1,47 @@
+{
+  buildGoModule,
+  fetchFromGitHub,
+  lib,
+  stdenvNoCC,
+  installShellFiles,
+  nix-update-script,
+  ...
+}:
+
+buildGoModule (finalAttrs: {
+  pname = "tcld";
+  version = "0.40.0";
+  src = fetchFromGitHub {
+    owner = "temporalio";
+    repo = "tcld";
+    rev = "refs/tags/v${finalAttrs.version}";
+    hash = "sha256-bIJSvop1T3yiLs/LTgFxIMmObfkVfvvnONyY4Bsjj8g=";
+  };
+  vendorHash = "sha256-GOko8nboj7eN4W84dqP3yLD6jK7GA0bANV0Tj+1GpgY=";
+  ldFlags = [
+    "-s"
+    "-w"
+  ];
+
+  # FIXME: Remove after https://github.com/temporalio/tcld/pull/447 lands.
+  patches = [ ./compgen.patch ];
```

Comment:
maybe it doesn't make sense to patch this in, but not having tab completion is really annoying and it's a one-line change so hopefully upstream just lands it?